metric_source/local/timer.go

```diff
@@ -0,0 +1,43 @@
+package local
+
+type Timer struct {
```

Comment:
godoc missed and for all pr :)

Reply:
Wrote godocs for all structures. Should i write it for public functions as well?